README.md

```diff
@@ -176,7 +176,7 @@ For an overview of the full workflow, see the [workflow diagram](./docs/workflow
 
 #### Install with Nvidia CUDA
 
-- For the best CUDA experience, installing vLLM is necessary to serve Safetensors format models.
+- For the best CUDA experience, installing vLLM is necessary to serve Safetensors format models. You can install using pip install or downloading CUDA from [Nvidia](https://developer.nvidia.com/cuda-downloads?target_os=Linux&target_arch=x86_64&Distribution=Ubuntu&target_version=22.04&target_type=deb_local)
```

Comment:
Should we give a link that is not specific to the distro used? https://developer.nvidia.com/cuda-downloads ?

Reply:
updated to general link